upgrade-tests.groovy

```diff
@@ -109,7 +109,7 @@ node('cico-workspace') {
 			timeout(time: 120, unit: 'MINUTES') {
 				upgrade_args = "--test-cephfs=false --test-rbd=true --upgrade-testing=true"
```

Comment:
```suggestion
				upgrade_args = "--test-cephfs=false --test-rbd=true --deploy-timeout=300 --upgrade-testing=true"
```

Reply:
well, rbd does not seem to fail that often